<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>691</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Cadastro</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>4</y>
        <width>691</width>
        <height>361</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="InserirDados">
           <property name="text">
            <string>Inserir Dados</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="OrdenarNome">
           <property name="text">
            <string>Ordenar por nome</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="OrdenarQtd">
           <property name="text">
            <string>Ordenar por quantidade</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="OrdenarValor">
           <property name="text">
            <string>Ordenar por valor</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Excluir">
           <property name="text">
            <string>Excluir Personalizado</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Personalizado</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Nome"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Quantidade</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Quantidade"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Valor</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Valor"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="tabela">
           <column>
            <property name="text">
             <string>Personalizado</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Qtd.Personalizado</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Valor unitário</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Valor total</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Valor total da conta:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Estatística</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>158</width>
        <height>261</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Produto com maior quantidade.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Produto com menor quantidade.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Produto com maior valor total.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Produto com menor valor total.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>724</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionSalvar"/>
    <addaction name="actionCarregar"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="actionCarregar">
   <property name="text">
    <string>Carregar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
